### PR TITLE
ICMSLST-2389 - Authority Expiring emails

### PR DESCRIPTION
**Pull request by marcuspp**

### CODE REVIEW

File: config/celery.py [hunk 1]
```diff
@@ -13,12 +15,12 @@
 
 app.conf.beat_schedule = {
     # Only enable when constabulary contacts manage firearms authorities
-    # "firearms_authority_expiring": {
-    #     "task": "web.notify.notify.send_firearms_authority_expiry_notification",
+    # "authority_expiring_firearms_email": {
```
Comment (marcuspp) [inline config/celery.py:18]: Left commented out as comment above still true

File: web/mail/emails.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+from datetime import date
```
Comment (MattHolmes123) [inline web/mail/emails.py:1]: Can you import this as `import datetime` or `import datetime as dt` please.
I have had problems in the past importing datetime / date directly from the datetime module.

Comment (MattHolmes123) [reply web/mail/emails.py:1]: there isn't 100% consistency in ICMS but the majority of imports import the whole module.

**Review by MattHolmes123 — APPROVED** [review]

File: web/mail/messages.py [hunk 3]
```diff
@@ -503,6 +507,62 @@ def get_site_domain(self) -> str:
...
+        self.authority_type = authority_type
+        self.expiry_date = expiry_date
+
+    def get_context(self) -> dict:
+        context = super().get_context()
+        return context | {
+            "importers_count": len(self.importers_details),
+            "authority_type": self.authority_type,
+            "summary_text": self.get_summary_text(),
+            "expiry_date": self.expiry_date.strftime("%-d %B %Y"),
```
Comment (mikeylockwood) [inline web/mail/messages.py:530]: We use this `"%-d %B %Y"` format in multiple places. That string should probably be a constant

**Review by mikeylockwood — APPROVED** [review]

**Pull request merged**